Header header
float64 pan_speed
float64 tilt_speed
---
bool result